common/src/main/java/org/apache/seata/common/DefaultValues.java

```diff
@@ -259,7 +259,7 @@ public interface DefaultValues {
     /**
      * the const DEFAULT_METRICS_ENABLED
      */
-    boolean DEFAULT_METRICS_ENABLED = false;
+    boolean DEFAULT_METRICS_ENABLED = true;
```

Comment:
Why is the default value changed to true?

Reply:
Monitoring data should be exposed from the kernel layer by default, and whether to display it is controlled by whether the service layer is connected to the corresponding monitoring system.